<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IntroWindow</class>
 <widget class="QMainWindow" name="IntroWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>1000</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>키오스크</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="BackButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>src/images/icons/arrow.png</normaloff>src/images/icons/arrow.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>560</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>원하시는 고기를 선택하세요.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>700</width>
      <height>10</height>
     </rect>
    </property>
    <property name="value">
     <number>40</number>
    </property>
   </widget>
   <widget class="QPushButton" name="SalmonButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>250</y>
      <width>250</width>
      <height>250</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>연어</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>src/images/assets/meat/salmon.png</normaloff>src/images/assets/meat/salmon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="HamButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>250</y>
      <width>250</width>
      <height>250</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>햄</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>src/images/assets/meat/ham.png</normaloff>src/images/assets/meat/ham.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="ChickenButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>500</y>
      <width>250</width>
      <height>250</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>닭가슴살</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>src/images/assets/meat/chicken_breast.png</normaloff>src/images/assets/meat/chicken_breast.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="SteakButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>500</y>
      <width>250</width>
      <height>250</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>스테이크</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>src/images/assets/meat/steak.png</normaloff>src/images/assets/meat/steak.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
